.github/workflows/integration-package-tests.yaml

```diff
@@ -18,6 +22,18 @@ jobs:
     # These tests will only run if the integration paths are affected, or someone has
     # added the `integrations` label to the PR
     runs-on: ubuntu-latest
+    if: |
+      github.event_name == 'push' ||
+      github.event_name == 'pull_request' &&
+      (
+        contains(github.event.pull_request.labels.*.name, 'test-all-integrations') ||
+        (
+          github.event.pull_request.changed_files &&
+          contains(github.event.pull_request.changed_files, '.github/workflows/integration-package-tests.yaml') ||
+          contains(github.event.pull_request.changed_files, 'src/integrations') ||
+          contains(github.event.pull_request.changed_files, 'src/**/*.py')
```

Comment:
I think we might want to leave this leg off because this will catch changes to any `src/prefect` files too